docs/network.md

```diff
@@ -0,0 +1,78 @@
+# 7. Adding new edges to routing tables
+
+This section covers the process of adding new edges, received from another nodes,
+to the routing table. It consists of several steps covered below.
+
+## 7.1 Step 1
+`PeerManagerActor` receives `RoutingTableSync` message containing list of new `edges` to add.
+This message is then forwarded to `RoutingTableActor`.
+Field `accounts` will be covered in a section 17. (TODO)
+
+```rust
+pub struct RoutingTableSync {
+    /// List of known edges from `RoutingTableActor::edges_info`.
+    edges: Vec<Edge>,
+    /// List of known `account_id` to `PeerId` mappings.
+    /// Useful for `send_message_to_account` method, to route message to particular account.
+    accounts: Vec<AnnounceAccount>,
+}
+```
+
+## 7.2 Step 2
+`PeerManagerActor` forwards those edges to `RoutingTableActor` inside of `ValidateEdgeList` struct.
+```rust
+pub struct ValidateEdgeList {
```

Comment:
this is too much detail - that will be hard to keep in sync. Instead try to describe the gist of the struct (if the name is  not enough).

Reply:
Ok, I made it simpler.